src/components/quests/QuestUpdated/QuestUpdated.tsx

```diff
@@ -0,0 +1,15 @@
+import { TimeAgo } from "@/components/TimeAgo";
+import type { Doc } from "@convex/_generated/dataModel";
+
+interface QuestUpdatedProps {
+  quest: Doc<"quests">;
+}
+export function QuestUpdated({ quest }: QuestUpdatedProps) {
+  const updatedTime = quest.updatedAt ? (
+    <TimeAgo date={new Date(quest.updatedAt)} />
+  ) : (
+    "Never"
```

Comment:
![CleanShot 2025-02-11 at 20 00 59@2x](https://github.com/user-attachments/assets/45943afd-4a75-4b69-8f55-396250123114)

This feels strange 🤔 

```suggestion
    "some time ago"
```

If the user is viewing the quests, it exists, so it's been updated at *some* point. Ideally they never see this state, but maybe a gradual degradation could be to say, "Updated some time ago" as a fallback?

Edit: if we make `updatedAt` a required field in the schema it should avoid this entirely.